src/java.base/share/classes/sun/nio/ch/SelectorProviderImpl.java

```diff
@@ -54,6 +54,12 @@ public DatagramChannel openUninterruptibleDatagramChannel() throws IOException {
         return new DatagramChannelImpl(this, /*interruptible*/false);
     }
 
+    public DatagramChannel openUninterruptibleDatagramChannel(ProtocolFamily family) throws IOException {
+    if(family == null)
```

Comment:
```suggestion
    if (family == null)
```

Reply:
Thank you @turbanoff for highlighting this.